index.html

```diff
@@ -361,6 +361,27 @@ <h3>P-521</h2>
         </pre>
       </section>
 
+      <section class="informative">
+        <h3>RSA</h3>
+
+        <section class="informative">
+          <h4>2048-bit</h4>
+          <p>These DID always start with <code>z4MX</code>.</p>
+          <pre class="example">
+            did:key:z4MXj1wBzi9jUstyPMS4jQqB6KdJaiatPkAtVtGc6bQEQEEsKTic4G7Rou3iBf9vPmT5dbkm9qsZsuVNjq8HCuW1w24nhBFGkRE4cd2Uf2tfrB3N7h4mnyPp1BF3ZttHTYv3DLUPi1zMdkULiow3M1GfXkoC6DoxDUm1jmN6GBj22SjVsr6dxezRVQc7aj9TxE7JLbMH1wh5X3kA58H3DFW8rnYMakFGbca5CB2Jf6CnGQZmL7o5uJAdTwXfy2iiiyPxXEGerMhHwhjTA1mKYobyk2CpeEcmvynADfNZ5MBvcCS7m3XkFCMNUYBS9NQ3fze6vMSUPsNa6GVYmKx2x6JrdEjCk3qRMMmyjnjCMfR4pXbRMZa3i
+          </pre>
+        </section>
+
+        <section class="informative">
+          <h4>4096-bit</h4>
+          <p>These DID always start with <code>zgg</code>.</p>
+          <pre class="example">
+            did:key:zgghBUVkqmWS8e1ioRVp2WN9Vw6x4NvnE9PGAyQsPqM3fnfPf8EdauiRVfBTcVDyzhqM5FFC7ekAvuV1cJHawtfgB9wDcru1hPDobk3hqyedijhgWmsYfJCmodkiiFnjNWATE7PvqTyoCjcmrc8yMRXmFPnoASyT5beUd4YZxTE9VfgmavcPy3BSouNmASMQ8xUXeiRwjb7xBaVTiDRjkmyPD7NYZdXuS93gFhyDFr5b3XLg7Rfj9nHEqtHDa7NmAX7iwDAbMUFEfiDEf9hrqZmpAYJracAjTTR8Cvn6mnDXMLwayNG8dcsXFodxok2qksYF4D8ffUxMRmyyQVQhhhmdSi4YaMPqTnC1J6HTG9Yfb98yGSVaWi4TApUhLXFow2ZvB6vqckCNhjCRL2R4MDUSk71qzxWHgezKyDeyThJgdxydrn1osqH94oSeA346eipkJvKqYREXBKwgB5VL6WF4qAK6sVZxJp2dQBfCPVZ4EbsBQaJXaVK7cNcWG8tZBFWZ79gG9Cu6C4u8yjBS8Ux6dCcJPUTLtixQu4z2n5dCsVSNdnP1EEs8ZerZo5pBgc68w4Yuf9KL3xVxPnAB1nRCBfs9cMU6oL1EdyHbqrTfnjE8HpY164akBqe92LFVsk8RusaGsVPrMekT8emTq5y8v8CabuZg5rDs3f9NPEtogjyx49wiub1FecM5B7QqEcZSYiKHgF4mfkteT2
```

Comment:
Please line wrap these key values. If you don't, the horizontal scrollbar for this spec is going to go on forever. We might want to add a CSS property for line-wrapping pre tags.

Reply:
The pre blocks have CSS to limit their width and add an internal horizontal scrollbar, rather than causing a horizontal scrollbar on the whole page. So there shouldn't be horizontal scrollbar on the page introduced here, unless the browser doesn't support the CSS that limits the width of the code blocks. To cause line wrapping for such browsers  (e.g. [Dillo](https://www.dillo.org/), [w3m](https://screenshots.debian.net/package/w3m)), I think it would require adding whitespace - which would break the ability to copy-paste the value, reducing functionality.

---

If you mean about the horizontal scrollbar in the code blocks, that would look like this currently:
![2021-11-20-191235_854x265_scrot](https://user-images.githubusercontent.com/95347/142744529-e0262529-f007-457a-9150-f484c20fa7c4.png)

P-521 has a horizontal scrollbar already. It has two example values:
![2021-11-20-191418_845x308_scrot](https://user-images.githubusercontent.com/95347/142744543-222c3932-6390-414f-bc33-f2fdc0c6ce09.png)

With `white-space: pre-wrap` on the `code` or `pre` tag, RSA 2048 would look like this:

![2021-11-20-191509_819x217_scrot](https://user-images.githubusercontent.com/95347/142744558-c4d6bfc0-ba70-462d-b162-95f5eb546503.png)

RSA 4096:
![2021-11-22-115536_817x294_scrot](https://user-images.githubusercontent.com/95347/142902764-aa56c620-f430-4e8b-8921-f9ebfe763da2.png)

And P-521 would look like this:

![2021-11-20-191434_824x198_scrot](https://user-images.githubusercontent.com/95347/142744550-5d31ac92-c5bf-4394-81dc-4f0a7c2f02d1.png)

Reply:
> With white-space: pre-wrap on the code or pre tag, RSA 2048 would look like this:

Yeah, let's do this for the time being -- can be done in a separate PR since I'm about to merge this one. As an aside, I've been wanting to add a re-spec extension to cleanly pre-wrap examples for a while now (and make a copy button that "unwraps/cleans" the example properly (remove comments, collapse multi-line into single line, etc.) when copied to the paste buffer so it's easy to paste it into other websites).